src/components/Todos/CreateTodo.js

```diff
@@ -0,0 +1,95 @@
+import React, { useState, useRef, Fragment, useContext } from 'react';
+
+import Card from '../UI/Card/Card';
+import classes from './CreateTodo.module.css';
+import Button from '../UI/Button/Button';
+import AuthContext from "../../store/AuthContext";
+
+const CreateTodoForm = () => {
+    const [isFormValid, setIsFormValid] = useState(false);
+    const [isShowForm, setShowForm] = useState(true);
+    const [message, setMessage] = useState(null);
+
+    const titleInput = useRef();
+    const descriptionInput = useRef();
+
+    const authCtx = useContext(AuthContext);
+    const token = authCtx.token;
+
+    const checkFomValidHandler= () => {
+        if (titleInput.current.value === "" ||
+            descriptionInput.current.value === "") {
+                setIsFormValid(false);
+                return;
+        }
+        setIsFormValid(true);
+    }
+
+    const submitHandler = (e) => {
+        e.preventDefault();
+        if (!isFormValid) {
+            return;
+        }
+        const createTodoAPI = 'http://localhost:5151/todos';
+        const createTodoData = {
+            title: titleInput.current.value,
+            description: descriptionInput.current.value,
+        };
+        fetch(createTodoAPI, {
+            method: "POST",
+            headers: {
+                'Content-Type': 'application/json',
+                'Authorization': token,
+            },
+            body: JSON.stringify(createTodoData),
+        })
+        .then(response => {
+            if (response.status === 200) {
+                setMessage("Signup successfully");
```

Comment:
It's page for creating Todos, you should change message and handle response after created.
(Example, you can redirect user to todos list after created)